<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="Yuki KATO" />
        <meta name="author" content="Yuki KATO" />
        <title>Yuki KATO, Ph.D.</title>
        <link rel="icon" type="image/x-icon" href="assets/k_icon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v6.3.0/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body>
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-light" id="mainNav">
            <div class="container px-4 px-lg-5">
                <a class="navbar-brand" href="index.html">Home</a>
                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
                    Menu
                    <i class="fas fa-bars"></i>
                </button>
                <div class="collapse navbar-collapse" id="navbarResponsive">
                    <ul class="navbar-nav ms-auto py-4 py-lg-0">
                        <li class="nav-item"><a class="nav-link px-lg-3 py-3 py-lg-4" href="about.html">About</a></li>
                        <li class="nav-item"><a class="nav-link px-lg-3 py-3 py-lg-4" href="research.html">Research</a></li>
                        <li class="nav-item"><a class="nav-link px-lg-3 py-3 py-lg-4" href="teaching.html">Teaching</a></li>
                    </ul>
                </div>
            </div>
        </nav>
        <!-- Page Header-->
        <header class="masthead" style="background-image: url('assets/img/bg-mountain.jpg')">
            <div class="container position-relative px-4 px-lg-5">
                <div class="row gx-4 gx-lg-5 justify-content-center">
                    <div class="col-md-10 col-lg-8 col-xl-7">
                        <div class="site-heading">
                            <h1>Research</h1>
                            <span class="subheading"><a class="nav-link" href="#publications">Publications</a></span>
                        </div>
                    </div>
                </div>
            </div>
        </header>
        <!-- Main Content-->
        <!-- Main Content-->
        <main class="mb-4">
            <div class="container px-4 px-lg-5">
                <div class="row gx-4 gx-lg-5 justify-content-center">
                    <div class="col-md-10 col-lg-8 col-xl-7">
                        <h2>Research Interests</h2>
                        <span class="subheading">bioinformatics, theoretical computer science, applied mathematics</span>
                        <h2>Main Topics</h2>
                        <ul>
                            <li><strong>Single-cell data analysis</strong><br>
                            (reported in websites of The University of Osaka <a href="https://www.ccb.osaka-u.ac.jp/en/wpccb_handle/wp-content/uploads/2024/03/KatoYuki2024EN.pdf">[Co-Creation Bureau]</a> in April 2024, <a href="https://resou.osaka-u.ac.jp/ja/research/2022/20221014_2">[ResOU]</a> and <a href="https://www.med.osaka-u.ac.jp/eng/activities/results/2022year/kato2022-10-17">[Graduate School of Medicine]</a> in October 2022, <a href="http://www.hpc.cmc.osaka-u.ac.jp/en/hpsc-news/vol08/">[Cybermedia Center]</a> in June 2022, and <a href="https://www.youtube.com/watch?v=yN1fkg2tjzA">YouTube</a> in June 2020)<br>
                            Please watch a <a href="https://ifremed.otri.osaka-u.ac.jp/member/mp4/Movie_Yuki_Kato.mp4">[video]</a> for further information.</li>
                            <li><strong>RNA structure analysis</strong><br>
                            (reported in websites of <a href="https://resou.osaka-u.ac.jp/ja/research/2021/20211006_2">[The University of Osaka]</a>, <a href="https://www.nikkei.com/article/DGXLRSP619040_W1A001C2000000/">[NIKKEI]</a>, <a href="https://bio.nikkeibp.co.jp/atcl/release/21/10/06/11695/">[Nikkei Biotechnology & Business]</a>, <a href="https://www.nikkan.co.jp/articles/view/00614965">[Nikkan]</a>, in October 2021; <a href="http://www.naist.jp/pressrelease/2012/08/002253.html">[Nara Institute of Science and Technology]</a>, <a href="http://news.mynavi.jp/news/2012/08/31/075/index.html">[Mynavi news]</a>, in August 2012; <a href="http://www.kuicr.kyoto-u.ac.jp/sites/icr/topics/topics_100922/">[Kyoto University]</a>, in September 2010)</li>
                            <li><strong>Sequence analysis</strong></li>
                            <li><strong>Formal language theory</strong></li>
                        </ul>
                        <p><a href="https://ifremed.otri.osaka-u.ac.jp/profile2023_11/index.html#page=21">生命とアルゴリズム: 生命現象をRNAデータとして見るためのものづくり</a></p>
                        <h2>Grants</h2>
                        <ol>
                            <li>日本医療研究開発機構研究費 脳神経科学統合プログラム (個別重点研究課題) 5-3「領域5 神経疾患・精神疾患の治療等のシーズ開発」ソロ型, 研究分担者, 高効率な点変異置換型RNA編集を可能にするアンチセンス核酸の開発, #24wm0625513h0001, 2024年8月&mdash;2027年3月.</li>
                            <li>科学研究費助成事業 (学術研究助成基金助成金) (基盤研究(B)), <strong>研究代表者</strong>, 細胞集団の時空間アラインメント法の確立, #24K03032, 2024年4月&mdash;2028年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-24K03032/">[KAKEN]</a></li>
                            <li>国立精神・神経医療研究センター 精神・神経疾患研究開発費, 研究分担者, 脳構成細胞の解析技術の基盤構築に基づく神経変性・修復機構の解明研究, #5-4, 2023年4月&mdash;2026年3月.</li>
                            <li>科学研究費助成事業 (科学研究費補助金) (基盤研究(B)), 研究分担者,  RNA編集不全に起因する自己免疫性脳症発症メカニムズの解明, #23H02576, 2023年4月&mdash;2026年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-23H02576/">[KAKEN]</a></li>
                            <li>科学研究費助成事業 (科学研究費補助金) (基盤研究(B)), 研究分担者, 修飾塩基を含むRNAの二次構造解析技術の確立, #22H03689, 2022年4月&mdash;2025年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-22H03689/">[KAKEN]</a></li>
                            <li>科学研究費助成事業 (学術研究助成基金助成金) (基盤研究(C)), <strong>研究代表者</strong>, 細胞動態の解明に向けた時系列細胞集団データ比較法の確立, #21K12109, 2021年4月&mdash;2025年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-21K12109/">[KAKEN]</a></li>
                            <li>科学研究費助成事業 (科学研究費補助金) (基盤研究(B)), 研究分担者, RNA編集酵素ADAR1の脳特異的機能の解明, #20H03341, 2020年4月&mdash;2023年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-20H03341/">[KAKEN]</a></li>                          
                            <li>日本医療研究開発機構研究費 難治性疾患実用化研究事業 (希少難治性疾患の克服に結びつく病態解明研究分野) (A-2), 研究分担者, 左巻きZ型RNAの認識機構破綻に着目したRNA編集酵素ADAR1変異型エカルディ・グティエール症候群 (6型AGS) 発症病態の解明, #20314695, 2020年4月&mdash;2023年3月.</li>
                            <li>日本医療研究開発機構研究費 難治性疾患実用化研究事業 (希少難治性疾患の克服に結びつく病態解明研究分野) (A-1), 研究分担者, TDP-43オリゴマーに着目したALSの病因解明と治療法の創出, #20314644, 2020年4月&mdash;2023年3月.</li>
                            <li>科学研究費助成事業 (科学研究費補助金) (基盤研究(B)), 研究分担者, ナノポアシークエンサーを用いたRNA二次構造決定法の開発, #19H04210, 2019年4月&mdash;2022年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-19H04210/">[KAKEN]</a></li>
                            <li>科学研究費助成事業 (学術研究助成基金助成金) (挑戦的研究(萌芽)), 研究分担者, マウス中枢神経組織における細胞選択的RNA発現プロファイル解析法の確立, #19K22580, 2019年6月&mdash;2021年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-19K22580/">[KAKEN]</a></li>
                            <li>科学研究費助成事業 (学術研究助成基金助成金) (基盤研究(C)), <strong>研究代表者</strong>, 種間で保存されたRNAグアニン４重鎖構造のゲノムワイド解析法の確立, #18K11526, 2018年4月&mdash;2023年3月. <a href="https://kaken.nii.ac.jp/grant/KAKENHI-PROJECT-18K11526/">[KAKEN]</a></li>
                            <li>科学研究費助成事業 (学術研究助成基金助成金) (基盤研究(C)), <strong>研究代表者</strong>, 構造情報の粗視化による高速ゲノムワイドRNA遺伝子発見, #15K00401, 2015年4月&mdash;2018年3月. <a href="https://kaken.nii.ac.jp/grant/KAKENHI-PROJECT-15K00401/">[KAKEN]</a></li>
                            <li>科学研究費助成事業 (学術研究助成基金助成金) (若手研究(B)), <strong>研究代表者</strong>, 高次構造を考慮した高速RNA間相互作用予測, #24700296, 2012年4月&mdash;2015年3月. <a href="https://kaken.nii.ac.jp/grant/KAKENHI-PROJECT-24700296/">[KAKEN]</a></li>                           
                            <li>科学研究費補助金 (挑戦的萌芽研究), 研究分担者, 機械学習と最適化に基づくRNAタンパク質相互作用予測, #23650153, 2011年4月&mdash;2013年3月. <a href="https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-23650153/">[KAKEN]</a></li>
                            <li>科学研究費補助金 (若手研究(B)), <strong>研究代表者</strong>, ハイブリッド型最適化によるRNA間相互作用予測, #22700313, 2010年4月&mdash;2012年3月. <a href="https://kaken.nii.ac.jp/grant/KAKENHI-PROJECT-22700313/">[KAKEN]</a></li>
                            <li>科学研究費補助金 (若手研究(スタートアップ)), <strong>研究代表者</strong>, 形式文法に基づくRNAタンパク質相互作用予測, #20800023, 2008年4月&mdash;2010年3月. <a href="https://kaken.nii.ac.jp/grant/KAKENHI-PROJECT-20800023/">[KAKEN]</a></li>
                            <li>科学研究費補助金 (特別研究員奨励費), <strong>研究代表者</strong>, 生物配列の高次構造記述向き形式文法とその構造予測への応用, #172830, 2005年4月&mdash;2008年3月. <a href="https://kaken.nii.ac.jp/grant/KAKENHI-PROJECT-05J02830/">[KAKEN]</a></li>
                        </ol>
                        <h2 id="publications">Publications</h2>
                        <h3>Journal Papers</h3>
                    <ol>
                        <li>Kyoka Higuchi, Akiko Uyeda, Lili Quan, Shogo Tanabe, <strong>Yuki Kato</strong>, Yukio Kawahara, and Rieko Muramatsu,<br>
                        Synaptotagmin 4 supports spontaneous axon sprouting after spinal cord injury,<br>
                        <em>The Journal of Neuroscience</em>, vol. 44, issue 43, e1593232024, Oct. 2024.
                        <a href="https://www.jneurosci.org/content/early/2024/09/11/JNEUROSCI.1593-23.2024.long">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/39266302/">[PubMed]</a></li>

                        <li>Kotaro Miyake, Pedro Henrique Costa Cruz, Izumi Nagatomo, <strong>Yuki Kato</strong>, Daisuke Motooka, Shingo Satoh, Yuichi Adachi, Yoshito Takeda, Yukio Kawahara, and Atsushi Kumanogoh,<br>
                        A cancer-associated METTL14 mutation induces aberrant m6A modification, affecting tumor growth,<br>
                        <em>Cell Reports</em>, vol. 42, issue 7, 112688, Jul. 2023.
                        <a href="https://doi.org/10.1016/j.celrep.2023.112688">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/37355987/">[PubMed]</a></li>

                        <li>Yanfang Xing, Taisuke Nakahama, Yuke Wu, Maal Inoue, Jung In Kim, Hiroyuki Todo, Toshiharu Shibuya, <strong>Yuki Kato</strong>, and Yukio Kawahara,<br>
                        RNA editing of AZIN1 coding sites is catalyzed by ADAR1 p150 after splicing,<br>
                        <em>Journal of Biological Chemistry</em>, vol. 299, issue 7, 104840, Jul. 2023.
                        <a href="https://www.jbc.org/article/S0021-9258(23)01868-9/fulltext">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/37209819/">[PubMed]</a></li>

                        <li>Reiichi Sugihara, <strong>Yuki Kato</strong>*, Tomoya Mori, and Yukio Kawahara,<br>
                        Alignment of single-cell trajectory trees with CAPITAL,<br>
                        <em>Nature Communications</em>, vol. 13, 5972, Oct. 2022 (*: corresponding author).
                        <a href="https://www.nature.com/articles/s41467-022-33681-3">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/36241645/">[PubMed]</a></li>

                        <li>Kengo Sato and <strong>Yuki Kato</strong>,<br>
                        Prediction of RNA secondary structure including pseudoknots for long sequences,<br>
                        <em>Briefings in Bioinformatics</em>, vol. 23, issue 1, bbab395, Jan. 2022.
                        <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab395/6380459?login=true">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/34601552/">[PubMed]</a></li>
                        
                        <li>Maal Inoue, Taisuke Nakahama, Ryuichiro Yamasaki, Toshiharu Shibuya, Jung In Kim, Hiroyuki Todo, Yanfang Xing, <strong>Yuki Kato</strong>, Eiichi Morii, and Yukio Kawahara,<br>
                        An Aicardi-Goutieres syndrome-causative point mutation in Adar1 gene invokes multi-organ inflammation and late-onset encephalopathy in mice,<br>
                        <em>The Journal of Immunology</em>, vol. 207, issue 12, pp. 3016-3027, Dec. 2021.
                        <a href="https://www.jimmunol.org/content/207/12/3016.long">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/34772697/">[PubMed]</a></li>
                        
                        <li>Akiko Uyeda, Quan Lili, <strong>Yuki Kato</strong>, Nagaaki Muramatsu, Shogo Tanabe, Kazuhisa Sakai, Noritaka Ichinohe, Yukio Kawahara, Tatsunori Suzuki, and Rieko Muramatsu,<br>
                        DDAH1 as a novel regulator of oligodendrocyte differentiation in the central nervous system remyelination,<br>
                        <em>GLIA</em>, vol. 69, issue 11, pp. 2591-2604, Nov. 2021.
                        <a href="https://onlinelibrary.wiley.com/doi/10.1002/glia.24060">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/34270117/">[PubMed]</a></li>
                        
                        <li>Taisuke Nakahama, <strong>Yuki Kato</strong>, Toshiharu Shibuya, Maal Inoue, Jung In Kim, Tuangtong Vongpipatana, Hiroyuki Todo, Yanfang Xing, and Yukio Kawahara,<br>
                        Mutations in the adenosine deaminase ADAR1 that prevent endogenous Z-RNA editing induce Aicardi-Goutieres syndrome-like encephalopathy,<br>
                        <em>Immunity</em>, vol. 54, issue 9, pp. 1976-1988, Sep. 2021.
                        <a href="https://www.cell.com/immunity/fulltext/S1074-7613(21)00356-3">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/34525338/">[PubMed]</a></li>

                        <li>Jung In Kim, Taisuke Nakahama, Ryuichiro Yamasaki, Pedro Henrique Costa Cruz, Tuangtong Vongpipatana, Maal Inoue, Nao Kanou, Yanfang Xing, Hiroyuki Todo, Toshiharu Shibuya, <strong>Yuki Kato</strong>, and Yukio Kawahara,<br>
                        RNA editing at a limited number of sites is sufficient to prevent MDA5 activation in the mouse brain,<br>
                        <em>PLoS Genetics</em>, vol. 17, issue 5, e1009516, May 2021.
                        <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009516">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/33983932/">[PubMed]</a></li>
                        
                        <li>Masumi Ito, Rieko Muramatsu, <strong>Yuki Kato</strong>, Bikram Sharma, Akiko Uyeda, Shogo Tanabe, Harutoshi Fujimura, Hiroyasu Kidoya, Nobuyuki Takakura, Yukio Kawahara, Masaki Takao, Hideki Mochizuki, Akiyoshi Fukamizu, and Toshihide Yamashita,<br>
                        Age-dependent decline in remyelination capacity is mediated by apelin-APJ signaling,<br>
                        <em>Nature Aging</em>, vol. 1, pp. 284-294, Mar. 2021.
                        <a href="https://www.nature.com/articles/s43587-021-00041-7">[Link]</a></li>
                        
                        <li>Keisuke Shimmura, <strong>Yuki Kato</strong>*, and Yukio Kawahara,<br>
                        Bivartect: accurate and memory-saving breakpoint detection by direct read comparison,<br>
                        <em>Bioinformatics</em>, vol. 36, issue 9, pp. 2725-2730, May 2020 (*: corresponding author).
                        <a href="https://doi.org/10.1093/bioinformatics/btaa059">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/31985791/">[PubMed]</a></li>

                        <li>Tuangtong Vongpipatana, Taisuke Nakahama, Toshiharu Shibuya, <strong>Yuki Kato</strong>, and Yukio Kawahara,<br>
                        ADAR1 regulates early T cell development via MDA5-dependent and -independent pathways,<br>
                        <em>The Journal of Immunology</em>, vol. 204, issue 8, pp. 2156-2168, Apr. 2020.
                        <a href="https://www.jimmunol.org/content/early/2020/03/12/jimmunol.1900929">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/32169840/">[PubMed]</a></li>
                        
                        <li>Pedro Henrique Costa Cruz, <strong>Yuki Kato</strong>, Taisuke Nakahama, Toshiharu Shibuya, and Yukio Kawahara,<br>
                        A comparative analysis of ADAR mutant mice reveals site-specific regulation of RNA editing,<br>
                        <em>RNA</em>, vol. 26, no. 4, pp. 454-469, Apr. 2020.
                        <a href="https://rnajournal.cshlp.org/content/26/4/454">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/31941663/">[PubMed]</a></li>
                        
                        <li>Taisuke Nakahama, <strong>Yuki Kato</strong>, Jung In Kim, Tuangtong Vongpipatana, Yutaka Suzuki, Carl Walkley, and Yukio Kawahara,<br>
                        ADAR1-mediated RNA editing is required for thymic self-tolerance and prevention of autoimmunity,<br>
                        <em>EMBO Reports</em>, vol. 19, issue 12, e46303, Dec. 2018.
                        <a href="http://embor.embopress.org/content/19/12/e46303">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/30361393/">[PubMed]</a></li>
                                
                        <li>Akino Nakasone, Rieko Muramatsu, <strong>Yuki Kato</strong>, Yukio Kawahara, and Toshihide Yamashita,<br>
                        Myotube-derived factor promotes oligodendrocyte precursor cell proliferation,<br/>
                        <em>Biochemical and Biophysical Research Communications</em>, vol. 500, issue 3, pp. 609-613, Jun. 2018.
                        <a href="https://www.sciencedirect.com/science/article/pii/S0006291X18308970?via%3Dihub">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/29679562/">[PubMed]</a></li>
                        
                        <li><strong>Yuki Kato</strong>*, Jan Gorodkin, and Jakob Hull Havgaard*,<br>
                        Alignment-free comparative genomic screen for structured RNAs using coarse-grained secondary structure dot plots,<br>
                        <em>BMC Genomics</em>, vol. 18, 935, Dec. 2017 (*: corresponding author).
                        <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-4309-y">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/29197323/">[PubMed]</a></li>
                        
                        <li><strong>Yuki Kato</strong>*, Tomoya Mori, Kengo Sato, Shingo Maegawa*, Hiroshi Hosokawa, and Tatsuya Akutsu,<br>
                        An accessibility-incorporated method for accurate prediction of RNA-RNA interactions from sequence data,<br>
                        <em>Bioinformatics</em>, vol. 33, no. 2, pp. 202-209, Jan. 2017 (*: corresponding author).
                        <a href="https://academic.oup.com/bioinformatics/article/33/2/202/2525711">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/27663495/">[PubMed]</a></li>
                        
                        <li>Michiaki Hamada, Yukiteru Ono, Hisanori Kiryu, Kengo Sato, <strong>Yuki Kato</strong>, Tsukasa Fukunaga, Ryota Mori, and Kiyoshi Asai,<br>
                        Rtools: a web server for various secondary structural analyses on single RNA sequences,<br>
                        <em>Nucleic Acids Research</em>, vol. 44, Web Server issue, pp. W302-W307, Jul. 2016.
                        <a href="http://nar.oxfordjournals.org/content/44/W1/W302">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/27131356/">[PubMed]</a></li>
                        
                        <li>Masato Yano and <strong>Yuki Kato</strong>*,<br>
                        Using hidden Markov models to investigate G-quadruplex motifs in genomic sequences,<br>
                        <em>BMC Genomics</em>, vol. 15(Suppl 9), S15, Dec. 2014 (*: corresponding author).
                        <a href="http://www.biomedcentral.com/1471-2164/15/S9/S15">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/25521044/">[PubMed]</a><br>
                        (This paper is also presented at the 13th International Conference on Bioinformatics (InCoB2014), Sydney, Australia, Aug. 2014.)</li>
                        
                        <li>Kengo Sato, <strong>Yuki Kato</strong>, Tatsuya Akutsu, Kiyoshi Asai, and Yasubumi Sakakibara,<br>
                        DAFS: simultaneous aligning and folding of RNA sequences via dual decomposition,<br>
                        <em>Bioinformatics</em>, vol. 28, no. 24, pp. 3218-3224, Dec. 2012.
                        <a href="http://bioinformatics.oxfordjournals.org/content/28/24/3218">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/23060618/">[PubMed]</a></li>
                        
                        <li><strong>Yuki Kato</strong>*, Kengo Sato, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        Rtips: fast and accurate tools for RNA 2D structure prediction using integer programming,<br>
                        <em>Nucleic Acids Research</em>, vol. 40, Web Server issue, pp. W29-W34, Jul. 2012 (*: corresponding author).
                        <a href="http://nar.oxfordjournals.org/content/40/W1/W29">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/22600734/">[PubMed]</a></li>
                        
                        <li>Unyanee Poolsap*, <strong>Yuki Kato</strong>*, Kengo Sato, and Tatsuya Akutsu,<br>
                        Using binding profiles to predict binding sites of target RNAs,<br>
                        <em>Journal of Bioinformatics and Computational Biology</em>, vol. 9, issue 6, pp. 697-713, Dec. 2011 (*: joint first and corresponding authors).
                        <a href="http://dx.doi.org/10.1142/S0219720011005628">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/22084009/">[PubMed]</a></li>
                        
                        <li>Kengo Sato*, <strong>Yuki Kato</strong>*, Michiaki Hamada, Tatsuya Akutsu, and Kiyoshi Asai,<br>
                        IPknot: fast and accurate prediction of RNA secondary structures with pseudoknots using integer programming,<br>
                        <em>Bioinformatics</em>, vol. 27, no. 13, pp. i85-i93, Jul. 2011 (*: joint first and corresponding authors).
                        <a href="http://bioinformatics.oxfordjournals.org/content/27/13/i85.abstract">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/21685106/">[PubMed]</a><br>
                        (This paper is also presented at 19th Annual International Conference on Intelligent Systems for Molecular Biology and 10th European Conference on Computational Biology (ISMB/ECCB2011), Vienna, Austria, Jul. 2011.)</li>
                        
                        <li><strong>Yuki Kato</strong>*, Kengo Sato*, Michiaki Hamada, Yoshihide Watanabe, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        RactIP: fast and accurate prediction of RNA-RNA interaction using integer programming,<br>
                        <em>Bioinformatics</em>, vol. 26, no. 18, pp. i460-i466, Sep. 2010 (*: joint first and corresponding authors).
                        <a href="http://bioinformatics.oxfordjournals.org/content/26/18/i460.abstract">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/20823308/">[PubMed]</a><br>
                        (This paper is also presented at the 9th European Conference on Computational Biology (ECCB2010), Ghent, Belgium, Sep. 2010.)</li>
                        
                        <li>Yusuke Ishida, <strong>Yuki Kato</strong>*, Liang Zhao, Hiroshi Nagamochi, and Tatsuya Akutsu,<br>
                        Branch-and-bound algorithms for enumerating treelike chemical graphs with given path frequency using detachment-cut,<br>
                        <em>Journal of Chemical Information and Modeling</em>, vol. 50, no. 5, pp. 934-946, May 2010 (*: corresponding author).
                        <a href="http://pubs.acs.org/doi/abs/10.1021/ci900447z">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/20415496/">[PubMed]</a></li>
                        
                        <li><strong>Yuki Kato</strong>*, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        Dynamic programming algorithms and grammatical modeling for protein beta-sheet prediction,<br>
                        <em>Journal of Computational Biology</em>, vol. 16, no. 7, pp. 945-957, Jul. 2009 (*: corresponding author).
                        <a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2008.0228">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/19580523/">[PubMed]</a></li>
                        
                        <li><strong>Yuki Kato</strong>*, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        A grammatical approach to RNA-RNA interaction prediction,<br>
                        <em>Pattern Recognition</em>, vol. 42, issue 4, pp. 531-538, Apr. 2009 (*: corresponding author).
                        <a href="http://dx.doi.org/10.1016/j.patcog.2008.08.004">[Link]</a></li>
                        
                        <li>Unyanee Poolsap, <strong>Yuki Kato</strong>*, and Tatsuya Akutsu,<br>
                        Prediction of RNA secondary structure with pseudoknots using integer programming,<br>
                        <em>BMC Bioinformatics</em>, vol. 10(Suppl 1), S38, Jan. 2009 (*: corresponding author).
                        <a href="http://www.biomedcentral.com/1471-2105/10/S1/S38">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/19208139/">[PubMed]</a><br>
                        (This paper is also published in <em>Proc.</em> the Seventh Asia Pacific Bioinformatics Conference (APBC2009), pp. 439-448, Beijing, China, Jan. 2009.)</li>
                        
                        <li>Hiroyuki Seki and <strong>Yuki Kato</strong>,<br>
                        On the generative power of multiple context-free grammars and macro grammars,<br>
                        <em>IEICE Transactions on Information and Systems</em>, vol. E91-D, no. 2, pp. 209-221, Feb. 2008.</li>
                        
                        <li><strong>Yuki Kato</strong>*, Hiroyuki Seki, and Tadao Kasami,<br>
                        RNA pseudoknotted structure prediction using stochastic multiple context-free grammar,<br>
                        <em>IPSJ Transactions on Bioinformatics</em>, vol. 47, no. SIG17 (TBIO1), pp. 12-21, Nov. 2006 (*: corresponding author).
                        <a href="https://www.jstage.jst.go.jp/article/imt/2/1/2_1_79/_article">[Link]</a><br>
                        2006年度バイオ情報学研究会論文賞受賞, 2007年度情報処理学会論文賞受賞, 2006年度IPSJ Digital Courier船井若手奨励賞受賞.</li>
                        
                        <li><strong>Yuki Kato</strong>*, Hiroyuki Seki, and Tadao Kasami,<br>
                        On the generative power of grammars for RNA secondary structure,<br>
                        <em>IEICE Transactions on Information and Systems</em>, vol. E88-D, no. 1, pp. 53-64, Jan. 2005 (*: corresponding author).</li>
                    </ol>
                    <h3>Refereed Conference Papers</h3>
                    <ol>
                        <li>Nobuyoshi Mizoguchi, <strong>Yuki Kato</strong>, and Hiroyuki Seki,<br>
                        A grammar-based approach to RNA pseudoknotted structure prediction for aligned sequences,<br>
                        <em>Proc.</em> the 1st IEEE International Conference on Computational Advances in Bio and medical Sciences (ICCABS2011), pp. 135-140, Orlando, Florida, USA, Feb. 2011.</li>
                        
                        <li>Unyanee Poolsap*, <strong>Yuki Kato</strong>*, and Tatsuya Akutsu,<br>
                        Dynamic programming algorithms for RNA structure prediction with binding sites,<br>
                        <em>Proc.</em> the Fifteenth Pacific Symposium on Biocomputing (PSB2010), vol. 15, pp. 98-107, The Big Island of Hawaii, USA, Jan. 2010 (*: joint first and corresponding authors).
                        <a href="http://psb.stanford.edu/psb-online/proceedings/psb10/poolsap.pdf">[Link]</a>
                        <a href="https://pubmed.ncbi.nlm.nih.gov/19908362/">[PubMed]</a></li>
                        
                        <li><strong>Yuki Kato</strong>*, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        Prediction of protein beta-sheets: dynamic programming versus grammatical approach,<br>
                        Third IAPR International Conference on Pattern Recognition in Bioinformatics (PRIB2008), <em>Lecture Notes in Bioinformatics</em>, vol. 5265, pp. 66-77, Melbourne, Australia, Oct. 2008 (*: corresponding author).
                        <a href="http://dx.doi.org/10.1007/978-3-540-88436-1_6">[Link]</a></li>
                        
                        <li><strong>Yuki Kato</strong>*, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        A grammatical approach to RNA-RNA interaction prediction,<br>
                        2007 International Symposium on Computational Models for Life Sciences (CMLS'07), <em>AIP Conference Proceedings</em>, vol. 952, pp. 197-206, Gold Coast, Australia, Dec. 2007 (*: corresponding author).</li>
                        
                        <li><strong>Yuki Kato</strong>*, Hiroyuki Seki, and Tadao Kasami,<br>
                        Stochastic multiple context-free grammar for RNA pseudoknot modeling,<br>
                        <em>Proc.</em> the 8th International Workshop on Tree Adjoining Grammar and Related Formalisms (TAG+8), pp. 57-64, Sydney, Australia, Jul. 2006 (*: corresponding author).</li>
                        
                        <li><strong>Yuki Kato</strong>*, Hiroyuki Seki, and Tadao Kasami,<br>
                        Subclasses of tree adjoining grammar for RNA secondary structure,<br>
                        <em>Proc.</em> the 7th International Workshop on Tree Adjoining Grammar and Related Formalisms (TAG+7), pp. 48-55, Vancouver, Canada, May 2004 (*: corresponding author).
                        <a href="https://www.aclweb.org/anthology/W04-3307/">[Link]</a></li>
                    </ol>
                    <h3>Books</h3>
                    <ol>
                        <li><strong>加藤 有己</strong>, 桜井 都衣, 藤渕 航,<br>
                        ヒト細胞からのビッグデータの情報管理と情報解析技術,<br>
                        <em>ビッグデータの収集、調査、分析と活用事例</em>, 第７章第２節, pp. 249-254, 技術情報協会, Nov. 2014.
                        <a href="http://www.gijutu.co.jp/doc/b_1775.htm">[Link]</a></li>
                        <li><strong>加藤 有己</strong>,<br>
                        二次構造に基づくRNA配列解析ソフトウェアの進展,<br>
                        有田 正規 編, 使えるデータベース・ウェブツール&mdash;日本発のデータベース戦略から，ゲノム・疾患情報の有効活用まで&mdash;, 第４章第８節, <em>実験医学</em>, vol. 29, no. 15 (増刊), pp. 2563-2570, 羊土社, Sep. 2011.
                        <a href="http://www.yodosha.co.jp/jikkenigaku/book/9784758103176/index.html">[Link]</a></li>
                    </ol>
                    <h3>Invited Talks</h3>
                    <ol>
                        <li><strong>加藤 有己</strong>,<br>
                        シングルセルデータを用いた細胞動態解析アルゴリズムの開発,<br>
                        NGS EXPO 2022, 大阪府大阪市, Oct. 2022.</li>
                        <li><strong>加藤 有己</strong>,<br>
                        複雑なRNA２次構造予測のための高速計算ソフトウェアの開発と今後の展開,<br>
                        次世代バイオインフォマティクス研究会, 北海道札幌市, Aug. 2011.</li>                       
                        <li><strong>Yuki Kato</strong>, Hiroyuki Seki, and Tatsuya Akutsu,<br>
                        RNA secondary structure prediction based on formal grammars,<br>
                        電子情報通信学会情報理論研究会, 電子情報通信学会技術研究報告, IT2007-21, 三重県志摩市, Nov. 2007.</li>
                    </ol>
                    <h3>Presentations at International Conferences</h3>
                    <ol>
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Jakob Hull Havgaard, and Yukio Kawahara,<br>
                        Deep learning-based prediction of potential RNA G-quadruplexes with D-Quartet,<br>
                        29th Conference on Intelligent Systems for Molecular Biology and the 20th European Conference on Computational Biology (ISMB/ECCB2021), Poster, 784, Virtual, Jul. 2021.</li>
                        
                        <li><strong>Yuki Kato</strong>, Reiichi Sugihara, Tomoya Mori, and Yukio Kawahara,<br>
                        Alignment of complex single-cell trajectories with CAPITAL,<br>
                        28th Conference on Intelligent Systems for Molecular Biology (ISMB2020), Poster, 725, Virtual, Jul. 2020.</li>
                        
                        <li><strong>Yuki Kato</strong>, Jan Gorodkin, and Jakob Hull Havgaard,<br>
                        Fast and efficient alignment-free comparative genomic screen for structured RNAs with DotcodeR,<br>
                        The 25th Conference on Intelligent Systems for Molecular Biology, and the 16th European Conference on Computational Biology (ISMB/ECCB2017), Poster, A-215, Prague, Czech Republic, Jul. 2017.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tomoya Mori, Kengo Sato, Shingo Maegawa, Hiroshi Hosokawa, and Tatsuya Akutsu,<br>
                        Accurate prediction of RNA-RNA interactions from sequence data incorporating interaction site accessibility,<br>
                        The 2016 joint annual meeting of the RNA Society and the RNA Society of Japan (RNA2016), Poster, 349, Kyoto, Japan, Jun. 2016.</li>
                        
                        <li>Michiaki Hamada, Yukiteru Ono, Hisanori Kiryu, Kengo Sato, <strong>Yuki Kato</strong>, Tsukasa Fukunaga, Ryota Mori, and Kiyoshi Asai,<br>
                        Rtools: a web server for various secondary structural analyses on single RNA sequences,<br>
                        The 2016 joint annual meeting of the RNA Society and the RNA Society of Japan (RNA2016), Poster, 342, Kyoto, Japan, Jun. 2016.</li>
                        
                        <li><strong>Yuki Kato</strong> and Wataru Fujibuchi,<br>
                        A computational method for dissecting single-cell populations based on transcriptomic data,<br>
                        CiRA/ISSCR 2016 International Symposium, Poster, P1-003, Kyoto, Japan, Mar. 2016.</li>
                        
                        <li><strong>Yuki Kato</strong>, Jakob Hull Havgaard, and Jan Gorodkin,<br>
                        Genomic screen for structured RNAs using coarse-grained dot plots,<br>
                        The First Joint Conference between GIW and InCoB (GIW/InCoB2015), Poster, 159, Tokyo, Japan, Sep. 2015.</li>
                        
                        <li><strong>Yuki Kato</strong>,<br>
                        De novo prediction of structured RNAs from genomes using coarse-grained dot plots,<br>
                        Talk at International Workshop on Computational Analysis of RNA Structure and Function, Benasque, Spain, Jul. 2015.</li>
                        
                        <li><strong>Yuki Kato</strong>, Jakob Hull Havgaard, and Jan Gorodkin,<br>
                        Using coarse-grained dot plots to screen for structurally similar RNAs in genomic sequences,<br>
                        The Thirteenth Asia Pacific Bioinformatics Conference (APBC2015), Poster, P-13, HsinChu, Taiwan, Jan. 2015.</li>
                        
                        <li>Kunie Sakurai, Junko Yamane, Kenta Kobayashi, Koji Yamanegi, Takeaki Taniguchi, <strong>Yuki Kato</strong>, and Wataru Fujibuchi,<br>
                        SHOGoiN cell database: a framework for a new repository on single cell assay data and diverse knowledge of human cells,<br>
                        The 18th Takeda Science Foundation Symposium on Bioscience, Poster, 070, Osaka, Japan, Jan. 2015.</li>
                        
                        <li>Kunie Sakurai, Junko Yamane, Kenta Kobayashi, Koji Yamanegi, Takeaki Taniguchi, <strong>Yuki Kato</strong>, and Wataru Fujibuchi,<br>
                        Stem Cell Informatics Database: a framework for a new repository on single cell assay data and diverse knowledge of human cells,<br>
                        The 25th International Conference on Genome Informatics (GIW/ISCB-Asia 2014), Poster, 91, Tokyo, Japan, Dec. 2014.</li>
                        
                        <li>Takeaki Taniguchi, <strong>Yuki Kato</strong>, Susumu Goto, and Wataru Fujibuchi,<br>
                        Development of a pipeline for analysis of meta- and single cell genomic sequences,<br>
                        The 25th International Conference on Genome Informatics (GIW/ISCB-Asia 2014), Poster, 41, Tokyo, Japan, Dec. 2014.</li>
                        
                        <li><strong>Yuki Kato</strong>, Jakob Hull Havgaard, and Jan Gorodkin,<br>
                        Fast RNA structural comparison using coarse-grained base-pairing probabilities,<br>
                        The 24th International Conference on Genome Informatics (GIW2013), Poster, 29, Biopolis, Singapore, Dec. 2013.</li>
                        
                        <li>Kengo Sato, <strong>Yuki Kato</strong>, Tatsuya Akutsu, Kiyoshi Asai, and Yasubumi Sakakibara,<br>
                        DAFS: simultaneous aligning and folding of RNA sequences via dual decomposition,<br>
                        International Symposium on Genome Science "Expanding Frontiers of Genome Science," Poster, P34, Tokyo, Japan, Jan. 2013.</li>
                        
                        <li><strong>Yuki Kato</strong>,<br>
                        RNA structural alignment using dual decomposition,<br>
                        Talk at Herbstseminar der Bioinformatik (2012), Doubice, Czech Republic, Oct. 2012.</li>
                        
                        <li><strong>Yuki Kato</strong>,<br>
                        Fast and accurate prediction of RNA-RNA interactions using integer programming,<br>
                        Talk at International Workshop on RNA, Benasque, Spain, Jul. 2012.</li>
                        
                        <li><strong>Yuki Kato</strong>,<br>
                        Fast and accurate prediction of RNA pseudoknotted structures using integer programming,<br>
                        Talk at International Workshop on RNA, Benasque, Spain, Jul. 2012.</li>
                        
                        <li>Kengo Sato, <strong>Yuki Kato</strong>, Michiaki Hamada, Tatsuya Akutsu, and Kiyoshi Asai,<br>
                        IPknot: fast and accurate prediction of RNA secondary structures with pseudoknots using integer programming,<br>
                        Sixteenth Annual Meeting of the RNA Society (RNA2011), Poster 803, Kyoto, Japan, Jun. 2011.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Michiaki Hamada, Yoshihide Watanabe, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        RactIP: fast and accurate prediction of RNA-RNA interaction using integer programming,<br>
                        Sixteenth Annual Meeting of the RNA Society (RNA2011), Oral 32, Poster 796, Kyoto, Japan, Jun. 2011.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Michiaki Hamada, Yoshihide Watanabe, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        RactIP: fast and accurate prediction of RNA-RNA interaction using integer programming,<br>
                        9th European Conference on Computational Biology (ECCB2010), Poster, C-47, Ghent, Belgium, Sep. 2010.</li>
                        
                        <li>Kengo Sato, <strong>Yuki Kato</strong>, Tatsuya Akutsu, and Kiyoshi Asai,<br>
                        IPknot: fast and accurate prediction of RNA secondary structures with pseudoknots using integer programming,<br>
                        9th European Conference on Computational Biology (ECCB2010), Poster, C-44, Ghent, Belgium, Sep. 2010.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Michiaki Hamada, Yoshihide Watanabe, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        RactIP: fast and accurate prediction of RNA-RNA interaction using integer programming,<br>
                        18th Annual International Conference on Intelligent Systems for Molecular Biology (ISMB2010), Poster, Boston, MA, USA, Jul. 2010.</li>
                        
                        <li>Unyanee Poolsap, <strong>Yuki Kato</strong>, and Tatsuya Akutsu,<br>
                        Dynamic programming algorithms for RNA structure prediction with binding sites,<br>
                        Poster and Software Demonstrations Book of the 20th International Conference on Genome Informatics (GIW2009), T001/P036, Yokohama, Japan, Dec. 2009.<br>
                        2009 Oxford Journals - Japanese Society for Bioinformatics Prize (GIW2009 Best Poster Award) 受賞.</li>
                        
                        <li>Nobuyoshi Mizoguchi, <strong>Yuki Kato</strong>, and Hiroyuki Seki,<br>
                        Pairwise RNA pseudoknotted structure prediction based on stochastic grammar,<br>
                        Poster and Software Demonstrations Book of the 20th International Conference on Genome Informatics (GIW2009), P018, Yokohama, Japan, Dec. 2009.</li>
                        
                        <li><strong>Yuki Kato</strong>,<br>
                        Prediction of interacting RNA secondary structures including binding sites,<br>
                        Talk at International Workshop on Computational methods for RNA analysis, Benasque, Spain, Aug. 2009.</li>
                        
                        <li><strong>Yuki Kato</strong>,<br>
                        RNA pseudoknot prediction based on integer programming,<br>
                        Talk at International Workshop on Computational methods for RNA analysis, Benasque, Spain, Jul. 2009.</li>
                        
                        <li>Unyanee Poolsap, <strong>Yuki Kato</strong>, and Tatsuya Akutsu,<br>
                        An integer programming-based method of predicting RNA secondary structure with pseudoknots and its relation to simple linear tree adjoining grammar,<br>
                        The 1st Thailand-Japan International Academic Conference 2008 (TJIA2008), Poster, Tokyo, Japan, Nov. 2008.</li>
                        
                        <li>Unyanee Poolsap, <strong>Yuki Kato</strong>, and Tatsuya Akutsu,<br>
                        Prediction of RNA secondary structure with pseudoknots using integer programming,<br>
                        Abstract Book of the 8th International Workshop on Bioinformatics and Systems Biology (IBSB2008), Berlin, Germany, Jun. 2008.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        A grammatical approach to RNA-RNA interaction prediction,<br>
                        Poster Abstracts of the Sixth Asia Pacific Bioinformatics Conference (APBC2008), page P79, Kyoto, Japan, Jan. 2008.</li>
                        
                        <li><strong>Yuki Kato</strong>, Keitaro Ichihara, and Yoshihide Watanabe,<br>
                        RNA-RNA interaction prediction based on integer programming,<br>
                        Poster Book of the 18th International Conference on Genome Informatics (GIW2007), pp. 70-71, Biopolis, Singapore, Dec. 2007.</li>
                        
                        <li><strong>Yuki Kato</strong>,<br>
                        Stochastic multiple context-free grammar for RNA pseudoknot modeling,<br>
                        Talk at International Workshop on Computational Approaches to Functional and Regulatory RNAs (RNA2006), Benasque, Spain, Jul. 2006.</li>
                        
                        <li><strong>Yuki Kato</strong>, Hiroyuki Seki, and Tadao Kasami,<br>
                        RNA structure prediction including pseudoknots based on stochastic multiple context-free grammar,<br>
                        Extended Abstract of International Workshop on Probabilistic Modeling and Machine Learning in Structural and Systems Biology (PMSB2006), pp. 32-37, Tuusula, Finland, Jun. 2006.</li>
                        
                        <li><strong>Yuki Kato</strong>, Hiroyuki Seki, and Tadao Kasami,<br>
                        A comparative study on formal grammars for pseudoknots,<br>
                        <em>Proc.</em> the 14th International Conference on Genome Informatics (GIW2003), pp. 470-471, Yokohama, Japan, Dec. 2003.</li>
                    </ol>
                    <h3>Presentations at Domestic Workshops</h3>
                    <ol>
                        <li>杉原 礼一, <strong>加藤 有己</strong>, 河原 行郎,<br>
                        グラフ埋め込みに基づく空間オミクス解析手法の開発,<br>
                        NGS EXPO 2024, P43, 大阪府大阪市, Sep. 2024.</li>
        
                        <li>Reiichi Sugihara, <strong>Yuki Kato</strong>, and Yukio Kawahara,<br>
                        A graph embedding approach to spatial omics analysis,<br>
                        第4回関西RNAクラブ, oral, 大阪府東大阪市, Aug. 2024.</li>
                        
                        <li>Reiichi Sugihara, <strong>Yuki Kato</strong>, and Yukio Kawahara,<br>
                        A random walk approach to cluster and integrate spatial transcriptomics data,<br>
                        第25回日本RNA学会年会, P-131, 東京都文京区, Jun. 2024.</li>
                        
                        <li>Hyebin Yoo, Taisuke Nakahama, <strong>Yuki Kato</strong>, and Yukio Kawahara,<br>
                        Elucidation of the pathogenic mechanism underlying Aicardi-Goutieres syndrome-like encephalopathy caused by mutations in the RNA-editing enzyme ADAR1,<br>
                        第25回日本RNA学会年会, O-13, 東京都文京区, Jun. 2024.</li>
        
                        <li>杉原 礼一, <strong>加藤 有己</strong>, 河原 行郎,<br>
                        グラフ埋め込みと最適輸送を用いた空間的遺伝子発現情報のアラインメント,<br>
                        第77回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2024-BIO-77 (21), 石川県能美市, Mar 2024.</li>
        
                        <li>杉原 礼一, <strong>加藤 有己</strong>, 森 智弥, 河原 行郎,<br>
                        1細胞RNA-seqデータを用いた細胞状態遷移経路アラインメント法の開発,<br>
                        RNA Frontier Meeting 2022, oral/poster, 大阪府吹田市, Oct. 2022.</li>
        
                        <li>Kengo Sato and <strong>Yuki Kato</strong>,<br>
                        Prediction of RNA secondary structure including pseudoknots for long sequences,<br>
                        第68回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2021-BIO-68 (5), Virtual, Nov. 2021.</li>
        
                        <li><strong>加藤 有己</strong>, 杉原 礼一, 森 智弥, 河原 行郎,<br>
                        CAPITALによるシングルセルデータの疑似時系列比較解析,<br>
                        第43回日本分子生物学会年会, 3LBA-066 (poster), Virtual, Dec. 2020.</li>
                        
                        <li>杉原 礼一, <strong>加藤 有己</strong>, 森 智弥, 河原 行郎,<br>
                        分岐を考慮した時系列シングルセルデータのアラインメント,<br>
                        第61回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2020-BIO-61 (6), 石川県能美市, Mar. 2020.</li>
                        
                        <li><strong>加藤 有己</strong>, 杉原 礼一, 森 智弥, 河原 行郎,<br>
                        複雑な細胞運命比較のための時系列１細胞RNA-seqデータのアラインメント,<br>
                        第42回日本分子生物学会年会, 1PW-14-2 (oral), 1P-0634 (poster), 福岡県福岡市, Dec. 2019.</li>
                        
                        <li><strong>加藤 有己</strong>, 佐藤 健吾, Jakob Hull Havgaard, 河原 行郎,<br>
                        畳み込みニューラルネットワークによるRNAグアニン４重鎖領域予測,<br>
                        第21回日本RNA学会年会, P-29, 東京都文京区, Jul. 2019.</li>
                        
                        <li>Pedro Henrique Costa Cruz, <strong>Yuki Kato</strong>, Taisuke Nakahama, Toshiharu Shibuya, and Yukio Kawahara,<br>
                        A comparative analysis among ADAR mutant mice reveals site-specific regulation of RNA editing,<br>
                        第21回日本RNA学会年会, P-5, 東京都文京区, Jul. 2019.</li>
                        
                        <li>Taisuke Nakahama, Tuangtong Vogpipatana, Toshiharu Shibuya, <strong>Yuki Kato</strong>, and Yukio Kawahara,<br>
                        ADAR1 regulates early T cell development via MDA5-dependent and -independent pathways,<br>
                        第21回日本RNA学会年会, O-3, 東京都文京区, Jul. 2019.</li>
                        
                        <li><strong>加藤 有己</strong>, 新村 啓介, 河原 行郎,<br>
                        配列リード直接比較による高精度省メモリゲノム構造変異解析,<br>
                        第41回日本分子生物学会年会, 1P-0792, 神奈川県横浜市, Nov. 2018.</li>
                        
                        <li><strong>加藤 有己</strong>, 佐藤 健吾, Jakob Hull Havgaard, 河原 行郎,<br>
                        深層学習に基づくRNAグアニン４重鎖構造識別法の検討,<br>
                        第20回日本RNA学会年会, P-40, 大阪府大阪市, Jul. 2018.</li>
                        
                        <li>新村 啓介, <strong>加藤 有己</strong>, 河原 行郎,<br>
                        ビット列のソートに基づくリード直接比較による高速省メモリゲノム構造変異解析,<br>
                        第53回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2018-BIO-53 (12), 石川県能美市, Mar. 2018.</li>
                        
                        <li>蒲原 純, <strong>加藤 有己</strong>, 河原 行郎,<br>
                        スペクトラルグラフ理論を応用した１細胞RNA-seqデータのトップダウン型クラスタリング,<br>
                        第53回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2018-BIO-53 (11), 石川県能美市, Mar. 2018.</li>
                        
                        <li>Takamasa Okanishi, <strong>Yuki Kato</strong>, Hiroyuki Fujimura, Yoshikatsu Nakano, Shoichiro Suda, Hideko Sone, and Wataru Fujibuchi,<br>
                        Network analysis of marine environmental factors based on MCMC sampling,<br>
                        生命医薬情報学連合大会2015年大会, Poster, P57, 京都府宇治市, Oct. 2015.</li>
                        
                        <li>Kenta Kobayashi, Midori Yuji, <strong>Yuki Kato</strong>, Takeaki Taniguchi, Toru Maruyama, Michihiro Ito, Susumu Goto, Haruko Takeyama, and Wataru Fujibuchi,<br>
                        Development of a pipeline for analysis of meta- and single-cell genomic sequences,<br>
                        生命医薬情報学連合大会2015年大会, Poster, P51, 京都府宇治市, Oct. 2015.</li>
                        
                        <li><strong>Yuki Kato</strong>, Jakob Hull Havgaard, and Jan Gorodkin,<br>
                        Genomic screen for structured RNAs using coarse-grained dot plots,<br>
                        生命医薬情報学連合大会2015年大会, Poster/Lightning Talk, P37 (F04), 京都府宇治市, Oct. 2015.</li>
                        
                        <li>小林 健太, <strong>加藤 有己</strong>, 谷口 丈晃, 丸山 徹, 伊藤 通浩, 五斗 進, 竹山 春子, 藤渕 航,<br>
                        海洋環境解析に向けたメタゲノムおよび１細胞配列データ解析用パイプラインの開発,<br>
                        第42回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2015-BIO-42 (58), 沖縄県国頭郡恩納村, Jun. 2015.</li>
                        
                        <li>岡西 孝真, <strong>加藤 有己</strong>, 藤村 弘行, 中野 義勝, 須田 彰一郎, 曽根 秀子, 藤渕 航,<br>
                        MCMCサンプリングに基づく海洋環境因子ネットワーク解析,<br>
                        第42回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2015-BIO-42 (35), 沖縄県国頭郡恩納村, Jun. 2015.</li>
                        
                        <li>桜井 都衣, 山根 順子, 小林 健太, 山根木 康嗣, 谷口 丈晃, <strong>加藤 有己</strong>, 藤渕 航,<br>
                        Stem Cell Informatics Database: 多様なヒト細胞情報の統合化を目指したデータベースの構築,<br>
                        第39回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2014-BIO-39 (5), 大阪府吹田市, Sep. 2014.</li>
                        
                        <li><strong>加藤 有己</strong>,<br>
                        ゲノムワイドRNA配列比較に向けて,<br>
                        2014年度RNAインフォマティクス道場, 北海道札幌市, Aug. 2014.</li>
                        
                        <li><strong>Yuki Kato</strong>, Jakob Hull Havgaard, and Jan Gorodkin,<br>
                        Fast RNA structural comparison using coarse-grained base-pairing probabilities,<br>
                        第16回日本RNA学会年会, Poster, P-91, 愛知県名古屋市, Jul. 2014.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tomoya Mori, Kengo Sato, Shingo Maegawa, Hiroshi Hosokawa, and Tatsuya Akutsu,<br>
                        Evaluating effectiveness of accessibility to infer RNA-RNA interactions,<br>
                        日本バイオインフォマティクス学会2013年年会, Poster, 68, 東京都江戸川区, Oct. 2013.</li>
                        
                        <li>Masato Yano and <strong>Yuki Kato</strong>,<br>
                        Modeling DNA G-quadruplexes by hidden Markov models,<br>
                        日本バイオインフォマティクス学会2013年年会, Poster, 6, 東京都江戸川区, Oct. 2013.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tomoya Mori, Kengo Sato, Shingo Maegawa, Hiroshi Hosokawa, and Tatsuya Akutsu,<br>
                        Evaluating effectiveness of accessibility to infer RNA-RNA interactions,<br>
                        第35回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2013-BIO-35 (3), 北海道札幌市, Sep. 2013.</li>
                        
                        <li><strong>加藤 有己</strong>,<br>
                        RNA間相互作用推定におけるアクセシビリティの有効性について,<br>
                        RNAアルゴリズム研究会2013, 東京都江東区, Aug. 2013.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        Rtips: fast and accurate tools for RNA 2D structure prediction using integer programming,<br>
                        第15回日本RNA学会年会, Poster, P-98, 愛媛県松山市, Jul. 2013.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tomoya Mori, Kengo Sato, Shingo Maegawa, Hiroshi Hosokawa, and Tatsuya Akutsu,<br>
                        Evaluating effectiveness of accessibility to infer RNA-RNA interactions,<br>
                        第15回日本RNA学会年会, Poster, P-93, 愛媛県松山市, Jul. 2013.</li>
                        
                        <li>佐藤 健吾, <strong>加藤 有己</strong>, 阿久津 達也, 浅井 潔, 榊原 康文,<br>
                        双対分解によるRNA構造アラインメント,<br>
                        第28回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2012-BIO-28 (5), 宮城県仙台市, Mar. 2012.<br>
                        2012年度情報処理学会山下記念研究賞受賞.</li>
                        
                        <li>Kengo Sato, <strong>Yuki Kato</strong>, Tatsuya Akutsu, and Kiyoshi Asai,<br>
                        Rtips: fast and accurate tools for RNA 2D structure prediction using integer programming,<br>
                        第34回日本分子生物学会年会, 一般講演/ポスター発表, 2T2a-5/2P-0112, 神奈川県横浜市, Dec. 2011.</li>
                        
                        <li>Kengo Sato, <strong>Yuki Kato</strong>, Tatsuya Akutsu, Kiyoshi Asai, and Yasubumi Sakakibara,<br>
                        Simultaneous aligning and folding of RNA sequences by dual decomposition,<br>
                        CBI/JSBi2011合同大会, Oral/Poster, JSBi-20, 兵庫県神戸市, Nov. 2011.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        Rtips: fast and accurate tools for RNA 2D structure prediction using integer programming,<br>
                        CBI/JSBi2011合同大会, Poster, JSBi-16, 兵庫県神戸市, Nov. 2011.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        Rtips: fast and accurate tools for RNA 2D structure prediction using integer programming,<br>
                        ICR Symposium to Celebrate the Bioinformatics Center's 10 Year Anniversary and New Restructuring, Poster, P03, 京都府宇治市, Aug. 2011.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Michiaki Hamada, Yoshihide Watanabe, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        RactIP: fast and accurate prediction of RNA-RNA interaction using integer programming,<br>
                        Computational Biology Research Center Workshop 2010 (CBRC2010), Poster, P44, 東京都江東区, Jul. 2010.</li>
                        
                        <li>佐藤 健吾, <strong>加藤 有己</strong>, 阿久津 達也, 浅井 潔,<br>
                        期待精度最大化に基づくRNAシュードノット予測,<br>
                        第22回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2010-BIO-22 (6), 東京都江東区, Jul. 2010.<br>
                        2011年度情報処理学会山下記念研究賞受賞.</li>
                        
                        <li><strong>Yuki Kato</strong>, Kengo Sato, Michiaki Hamada, Yoshihide Watanabe, Kiyoshi Asai, and Tatsuya Akutsu,<br>
                        RNA-RNA interaction prediction using integer programming with threshold cut,<br>
                        第21回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2010-BIO-21 (32), 沖縄県中頭郡西原町, Jun. 2010.</li>
                        
                        <li>Unyanee Poolsap, <strong>Yuki Kato</strong>, and Tatsuya Akutsu,<br>
                        Prediction of RNA secondary structures with binding sites using dynamic programming algorithm,<br>
                        第17回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2009-BIO-17 (13), 沖縄県国頭郡恩納村, May 2009.</li>
                        
                        <li>田中 翔, <strong>加藤 有己</strong>, 関 浩之,<br>
                        ペア確率多重文脈自由文法によるシュードノットつきRNA2次構造予測,<br>
                        第16回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2009-BIO-16 (10), 東京都文京区, Mar. 2009.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        Dynamic programming versus grammatical approach for protein beta-sheet prediction,<br>
                        The 2008 Annual Conference of the Japanese Society for Bioinformatics (JSBi2008), Poster, P023, 大阪府豊中市, Dec. 2008.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        Prediction of protein beta-sheets: dynamic programming versus grammatical approach,<br>
                        第13回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2008-BIO-13 (14), 沖縄県中頭郡西原町, Jun. 2008.</li>
                        
                        <li>Unyanee Poolsap, <strong>Yuki Kato</strong>, and Tatsuya Akutsu,<br>
                        Prediction of RNA secondary structure with pseudoknots using integer programming,<br>
                        第11回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2007-BIO-11 (32), 東京都江東区, Dec. 2007.</li>
                        
                        <li><strong>Yuki Kato</strong>, Tatsuya Akutsu, and Hiroyuki Seki,<br>
                        Secondary structure prediction of interacting RNAs: a grammatical approach,<br>
                        第11回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2007-BIO-11 (31), 東京都江東区, Dec. 2007.</li>
                        
                        <li>市原 慶太郎, <strong>加藤 有己</strong>, 渡邊 芳英,<br>
                        整数計画法に基づくRNA間相互作用予測,<br>
                        第11回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2007-BIO-11 (30), 東京都江東区, Dec. 2007.</li>
                        
                        <li>Hiroyuki Seki and <strong>Yuki Kato</strong>,<br>
                        On the generative power of multiple context-free grammars and macro grammars,<br>
                        電子情報通信学会コンピュテーション研究会, 電子情報通信学会技術研究報告, COMP2006-38, 宮城県仙台市, Oct. 2006.</li>
                        
                        <li><strong>加藤 有己</strong>,<br>
                        確率文法と生物配列解析への応用,<br>
                        第17回代数, 論理, 幾何と情報科学研究集会 (ALGI17), 奈良県奈良市, Oct. 2006.</li>
                        
                        <li><strong>Yuki Kato</strong>, Hiroyuki Seki, and Tadao Kasami,<br>
                        RNA pseudoknotted structure prediction using stochastic multiple context-free grammar,<br>
                        第6回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2006-BIO-6 (5), 奈良県生駒市, Sep. 2006.</li>
                        
                        <li><strong>Yuki Kato</strong> and Hiroyuki Seki,<br>
                        Stochastic multiple context-free grammar for RNA structure analysis,<br>
                        第3回情報処理学会バイオ情報学研究会, 情報処理学会研究報告, 2005-BIO-3 (2), 東京都港区, Dec. 2005.</li>
                        
                        <li><strong>Yuki Kato</strong>, Hiroyuki Seki, and Tadao Kasami,<br>
                        On the generative power of grammars for RNA secondary structure,<br>
                        電子情報通信学会コンピュテーション研究会, 電子情報通信学会技術研究報告, COMP2003-75, 東京都文京区, Jan. 2004.</li>
                        
                        <li>渡邊 芳英, 野田 良純, <strong>加藤 有己</strong>, 池上 大介,<br>
                        Asirによるトーリックイデアルのグレブナー基底の計算と整数計画問題,<br>
                        日本数式処理学会第11回大会, 東京都調布市, Sep. 2002.</li>
                    </ol>
                    </div>
                </div>
            </div>
        </main>
        <!-- Footer-->
        <footer class="border-top">
            <div class="container px-4 px-lg-5">
                <div class="row gx-4 gx-lg-5 justify-content-center">
                    <div class="col-md-10 col-lg-8 col-xl-7">
                        <ul class="list-inline text-center">
                            <li class="list-inline-item">
                                <a href="https://github.com/ykat0">
                                    <span class="fa-stack fa-lg">
                                        <i class="fas fa-circle fa-stack-2x"></i>
                                        <i class="fab fa-github fa-stack-1x fa-inverse"></i>
                                    </span>
                                </a>
                            </li>
                        </ul>
                        <div class="small text-center text-muted fst-italic">Copyright &copy; Yuki KATO's Website 2025</div>
                    </div>
                </div>
            </div>
        </footer>
        <!-- Bootstrap core JS-->
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
